### PR TITLE
Don't call close_tasks in begin_standby

### PR DESCRIPTION
**Pull request by pkubanek**

### CODE REVIEW

File: python/lsst/ts/MTAirCompressor/aircompressor_csc.py [hunk 1]
```diff
@@ -241,6 +244,7 @@ async def disconnect(self):
 
     async def end_start(self, data):
         """Enables communication with the compressor."""
+        print("end start 1")
```
Comment (tribeiro) [inline python/lsst/ts/MTAirCompressor/aircompressor_csc.py]: Consider using `self.log.debug` instead...

File: python/lsst/ts/MTAirCompressor/aircompressor_csc.py [hunk 2]
```diff
@@ -257,15 +261,17 @@ async def end_start(self, data):
             await self.connect()
             if self.poll_task.done():
                 self.poll_task = asyncio.create_task(self.poll_loop())
+            print("end start 2")
```
Comment (tribeiro) [inline python/lsst/ts/MTAirCompressor/aircompressor_csc.py]: consider using `self.log.debug`.

File: python/lsst/ts/MTAirCompressor/aircompressor_csc.py [hunk 3]
```diff
@@ -257,15 +261,17 @@ async def end_start(self, data):
...
         except (
             pymodbus.exceptions.ModbusException,
             asyncio.TimeoutError,
         ) as ex:
+            print("end start ex")
```
Comment (tribeiro) [inline python/lsst/ts/MTAirCompressor/aircompressor_csc.py]: consider using `self.log.debug`.

File: python/lsst/ts/MTAirCompressor/aircompressor_csc.py [hunk 4]
```diff
@@ -257,15 +261,17 @@ async def end_start(self, data):
...
             await self.log_modbus_exception(ex, "Starting up:", True)
             return
 
     async def begin_standby(self, data):
-        await self.close_tasks()
+        await self._close_own_tasks()
```
Comment (tribeiro) [inline python/lsst/ts/MTAirCompressor/aircompressor_csc.py:271]: consider using `self.log.debug`.

**Review by tribeiro — APPROVED** [review]
Looks good.. I left a few comments inline, mainly, consider using `self.log.debug` instead of `print`.

You can see log statements when running `pytest` if you do:

```
pytest --log-cli-level DEBUG
```

Also, consider start curating a version history for this package.

File: python/lsst/ts/MTAirCompressor/aircompressor_csc.py (hunk 1 above)
Comment (pkubanek) [reply python/lsst/ts/MTAirCompressor/aircompressor_csc.py]: That's a leftover from debugging..didn't notice it. Also all others. Print statements were removed.

**Review by r-owen — COMMENTED** [review]
This seems like a reasonable solution to me.

If you already have a `handle_summary_state`, or if you also want to close these tasks in FAULT state, then consider calling `_close_own_tasks` from `handle_summary_state`, instead of adding the `begin_standby` method. But barring either of those, overriding `begin_standby` is likely the cleanest solution.

**Pull request merged**